src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html

```diff
@@ -118,17 +118,45 @@
   </div>
   <div class="section-body">
     <table>
-      <tr>
-        <td></td>
-        <td>{{ ccj.claimantName}}</td>
-      </tr>
+      {% if ccj.signerName is empty %}
+        <tr>
+          <td>
+            Name:
+          </td>
+          <td>{{ ccj.claimantName}}</td>
```

Comment:
this doesn't look right

Reply:
?????

Reply:
signerName

Reply:
signerName is SOT Signer name, it is not same as claimant name, which we are displaying separately as signer name

Reply:
sorry this section is fine i missed the `is empty check`